data/tlg0643/tlg001/tlg0643.tlg001.1st1K-mul1.xml

```diff
@@ -86,7 +86,7 @@
     <encodingDesc>
       <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
       <refsDecl n="CTS">
-        <cRefPattern n="paragraph" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:p[@n='$1'])"/>

```

Comment:
This actually looked correct to me. The index is a type "commentary" and has only paragraphs marked up as `<p n="1">` etc.